include/powsybl/iidm/Branch.hpp

```diff
@@ -11,12 +11,19 @@
 #include <iosfwd>
 
 #include <powsybl/iidm/Connectable.hpp>
-#include <powsybl/iidm/CurrentLimitsAdder.hpp>
+#include <powsybl/iidm/CurrentLimits.hpp>
+#include <powsybl/iidm/OperationalLimitsHolder.hpp>
 
 namespace powsybl {
 
 namespace iidm {
 
+class ActivePowerLimits;
+class ActivePowerLimitsAdder;
```

Comment:
Use forward declaration for adders here because :
`ActivePowerLimitsAdder` needs `LoadingLimitsAdder<L, A>`, which needs `OperationalLimitsHolder`, which needs `ActivePowerLimitsAdder`